lib/parse.js

```diff
@@ -1320,11 +1320,17 @@ function parse($TEXT, options) {
             var type = start.type;
             var name = as_property_name();
             if (type == "name" && !is("punc", ":")) {
+                var createAccessor = function() {
```

Comment:
I tried both, first what you wrote, then this - I did not like executing the `createAccessor`outside of the get / set scopes, but it probably makes no difference (accessor must be `get` or `set` eventually).